packages/next-playwright/src/next-fixture.ts

```diff
@@ -0,0 +1,64 @@
+import type { BrowserContext } from '@playwright/test'
+import type { NextWorkerFixture } from './next-worker-fixture'
+
+type MockFetch = (request: Request) => Response | Promise<Response>
+type MockFetchJson = (request: Request) => any | Promise<any>
+
+export interface NextFixture {
+  mockFetch: (url: string, mock: MockFetch) => void
+  mockFetchJson: (url: string, json: any | MockFetchJson) => void
+}
+
+class NextFixtureImpl implements NextFixture {
+  public testId: string
+  private worker: NextWorkerFixture
+
+  constructor(nextWorker: NextWorkerFixture) {
+    this.worker = nextWorker
+    this.testId = Math.random().toString(36).replace('0.', '')
```

Comment:
Could we use Playwright's test IDs here? You can get that from `test.info()` in Playwright.

Reply:
TIL. Will try that.